boards/arm/ks_zion/ks_zion_defconfig

```diff
@@ -0,0 +1,26 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_STM32F3X=y
+CONFIG_SOC_STM32F303X8=y
+
+# Enable MPU
+CONFIG_ARM_MPU=n
```

Comment:
Symbols are generally defaulting to n.
You don't need to add anything to set it to n.

Reply:
ok